### DISCUSSION TITLE
databend create the snapshot for the table  as the data changed, But how to remove the data which are too old.

### DISCUSSION THREAD

**Discussion by xiaoshao** [databendlabs/databend#8498] [Q&A] [answered]
The data is changing and the snapshot & segments & blocks are growing.  One day some snapshot is usless.

How to remove it ?

**Comment by bohutang** [comment, marked as answer]
`optimize table <your-table> purge` will remove the snapshot files(only keep the latest one):
https://databend.rs/doc/reference/sql/ddl/table/optimize-table
